agent/lib/sdk.js

```diff
@@ -70,7 +77,7 @@ const createSDK = (emitter) => {
 
   const auth = async () => {
     if (config["TD_API_KEY"]) {
-      const url = [root, "auth/exchange-api-key"].join("/");
+      const url = [config["TD_API_ROOT"], "auth/exchange-api-key"].join("/");
```

Comment:
Sometimes people (me) accidentally leave a trailing `/` on the `TD_API_ROOT`.

For URLs, I've been reaching for `new URL` more often:
```suggestion
      const url = new URL("auth/exchange-api-key", config["TD_API_ROOT").href
      const url = [config["TD_API_ROOT"], "auth/exchange-api-key"].join("/");
```

> 'https://v6.testdriver.ai/auth/exchange-api-key'